cime_config/config_compsets.xml

```diff
@@ -326,6 +326,12 @@
     <science_support grid="f09_tn14"/>
     <science_support grid="f19_tn14"/>
   </compset>
+  <compset>
+    <alias>NHISTNCPM</alias>
+    <lname>HIST_CAM60%NORESM_CLM50%BGC-CROP_CICE%NORESM-CMIP6_BLOM%ECO_MOSART_SGLC_SWAV_NORCPM_BGC%BDRDDMS</lname>
+    <science_support grid="f09_tn14"/>
+    <science_support grid="f19_tn14"/>
```

Comment:
Are both of these resolutions scientifically supported for NorCPM?

Reply:
Not yet. I can remove them for now and add them back after validations.